src/number_theory/bernoulli_polynomials.lean

```diff
@@ -7,6 +7,9 @@ import data.polynomial.algebra_map
 import data.polynomial.derivative
 import data.nat.choose.cast
 import number_theory.bernoulli
+import analysis.special_functions.integrals
```

Comment:
Until now, this file was having only pretty low-level imports. You are bumping this much higher, making any number-theory stuff depending on Bernoulli polynomials also depend on measure theory, which is not desirable. Instead, could you create a new file like `bernoulli_polynomials_integral.lean` with the imports you want?